drivers/sensor/lm75/lm75.c

```diff
@@ -82,7 +89,13 @@ static int lm75_sample_fetch(const struct device *dev,
 		ret = -ENOTSUP;
 		break;
 	}
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	int ret2 = pm_device_runtime_put(dev);
 
+	if (ret2 < 0) {
+		LOG_ERR("pm_device_runtime_put failed with error %i", ret2);
+	}
+#endif
```

Comment:
what's going on with ret2?

Reply:
to check for that error and log it.. using ret would prevent returning the correct error code of the operation

Reply:
@gmarull I changed it a little bit, is this better?